lib/src/crop.dart

```diff
@@ -71,11 +71,13 @@ class _CropState extends State<Crop> with TickerProviderStateMixin {
   final _repaintBoundaryKey = GlobalKey();
 
   double _previousScale = 1;
+  double scaleLimit = 3; //default value 3
```

Comment:
This field can be passed through the constructor (in Crop class not in CropState).

Reply:
I suggest `maxZoom` for the field name. It also should be nullable with default value of `null`.

Reply:
I was not sure if it's okay to change the API of the Crop class. Will apply changes soon!